lib/cosmos/tools/cmd_tlm_server/api.rb

```diff
@@ -597,22 +597,26 @@ def inject_tlm(target_name, packet_name, item_hash = nil, value_type = :CONVERTE
         # Write to routers
         if send_routers
           router = CmdTlmServer.instance.routers.all['PREIDENTIFIED_ROUTER']
```

Comment:
The INSTALL configuration system.txt declares CTS_PREIDENTIFIED so why is PREIDENTIFIED_ROUTER not defined? Are we missing something in the specs for this? There is an api_spec ...

Reply:
Its just an order of operations issue.  In CmdTlmServer#start the preidentified router hasn't been added yet when then meta_callback is called.